packages/ra-core/src/form/choices/ChoicesContext.ts

```diff
@@ -0,0 +1,40 @@
+import { createContext } from 'react';
+import { FilterPayload, RaRecord, SortPayload } from '../../types';
+
+/**
+ * Context to store choices and functions to retrieve them.
+ *
+ * Use the useChoicesContext() hook to read the context.
+ */
+export const ChoicesContext = createContext<ChoicesContextValue>(undefined);
+
+export type ChoicesContextValue<RecordType extends RaRecord = any> = {
+    sort: SortPayload;
```

Comment:
could you please sort the keys alphabetically, or add comments to explain the sort logic?